internal/config/config.go

```diff
@@ -212,6 +251,16 @@ func Default() *Config {
 		StateDiskSizeGB: 30,
 		DebugCluster:    func() *bool { b := false; return &b }(),
 		Provider: ProviderConfig{
+			AWS: &AWSConfig{
+				Region:                 "",
+				Image:                  "",
+				InstanceType:           "t3.xlarge",
```

Comment:
The default instance type does not support nitro tpm:

```
Error: waiting for Auto Scaling Group (malte-8bc50119-control-plane) capacity satisfied: 1 error occurred:
	* Scaling activity (fad60f97-3445-a48c-c220-6b9716070b1a): Failed: The t3.xlarge instance type does not support an AMI with TPM version v2.0. Specify an instance type that supports TPM version v2.0, and try again. Launching EC2 instance failed.
```

Reply:
I searched for Amazon Nitro instances but I guess it turns out that AWS Nitro ≠ NitroTPM support:
 
https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/enable-nitrotpm-prerequisites.html

What a god awful naming scheme for Amazon’s product lineup. Absolute lunacy by Amazon branding this as Nitro yet supporting it on a limited subset of Nitro machines.

Will have to adjust the whole compatibility list later. Thanks Amazon, I hate it.

Reply:
Fixed. At least it has the benefit that the supported list is now way smaller and thus the check is way easier :)